src/obfl-specification.html

```diff
@@ -1557,18 +1559,43 @@ <h4 id="L735" class="include">table-of-contents</h4>
       </dl>
     </dd>
   <dt>Content Model</dt>
-    <dd>toc-entry [1 or more]</dd>
+    <dd>toc-block [1 or more]</dd>
+</dl>
+</div>
+
+<h4 id="L735" class="include">toc-block</h4>
+
+<p>The toc-block element defines a block of entries that can be used within a
+table of contents. A toc-block without descendant entries will not be rendered.</p>
```

Comment:
It is not so clear from this sentence that a toc-block with descendant toc-entry elements is not rendered when none of these toc-entry elements are rendered. If you read the sentence you might think that a toc-block is not rendered only when it has no descendant toc-entry elements.

Reply:
I have improved the explanation.

Reply:
OK looks better.